packages/app/cypress/e2e/top-nav.cy.ts

```diff
@@ -65,7 +65,6 @@ describe('App Top Nav Workflows', () => {
       it('shows the current browser in the top nav browser list button', () => {
         cy.findByTestId('top-nav-active-browser-icon')
         .should('have.attr', 'src')
-        .and('contain', 'chrome')
```

Comment:
no longer applicable since the src doesn't contain chrome when transformed by updated `vite-svg-loader`

Reply:
because chrome in this case is the active browser. the `src` no longer contains chrome so we can't assert on it, but the below assertion I feel should be enough